app/assets/stylesheets/base/reset.scss

```diff
@@ -0,0 +1,137 @@
+// TODO: uncomment when we're ready!
+// *,
+// *::before,
+// *::after {
+//   box-sizing: border-box;
```

Comment:
This bad boy is waiting here for the right moment to shine :)

Reply:
ahhaha what do you mean?

Reply:
@rhymes, the default `box-sizing` value is `content-box`. When you add e.g. padding to an element, it will be bigger than the width/height set. `box-sizing: border-box` takes those things into account which means you hvae more reliable sizing of elements.

See [box-sixing](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) on MDN.